lib/upipe-dvbcsa/libupipe_dvbcsa.pc.in

```diff
@@ -6,5 +6,5 @@ Name: libupipe_dvbcsa
 Description: Upipe multimedia framework, libdvbcsa interface module
 Version: @VERSION@
 Requires: libupipe libupipe_ts
-Libs: -L${libdir} -lupipe_dvbcsa
+Libs: -L${libdir} -lupipe_dvbcsa -ldvbcsa
```

Comment:
Shouldn't it be added to Libs.private instead?

Reply:
Maybe, will check the pkg-config docs

Reply:
Seems right, Libs.private content only appears if using pkg-config --static though